src/main/java/com/minecolonies/api/colony/requestsystem/requestable/Tool.java

```diff
@@ -177,7 +178,15 @@ public boolean matches(@NotNull final ItemStack stack)
             return false;
         }
 
-        return ItemStackUtils.hasEquipmentLevel(stack, getEquipmentType(), getMinLevel(), getMaxLevel());
+        try
+        {
+            return ItemStackUtils.hasEquipmentLevel(stack, getEquipmentType(), getMinLevel(), getMaxLevel());
+        }
+        catch (final Exception e)
+        {
+            Log.getLogger().warn("Got exception for Itemstack when trying to match: " +  stack.getDisplayName().getString(), e);
```

Comment:
print the full stack and mention that it is an error during getting equipment level^^